pubtools/_quay/iib_operations.py

```diff
@@ -104,7 +104,7 @@ def task_iib_add_bundles(
     _, tag = build_details.index_image.split(":", 1)
     dest_image = image_schema_tag.format(
         host=target_settings.get("quay_host", "quay.io").rstrip("/"),
-        namespace=target_settings["quay_namespace"],
+        namespace=target_settings.get("quay_operator_namespace", target_settings["quay_namespace"]),
```

Comment:
One day all repos will be in the same namespace, and at that time quay_operator_namespace won't be needed. Also "pending" target doesn't use different namespace, so that's why the fallback